src/main/java/org/repodriller/scm/SCM.java

```diff
@@ -68,7 +68,7 @@
 	Commit getCommit(String id);
 	/* TODO A method named getCommitXYZ should return a Commit. */
 	String getCommitFromTag(String tag);
-
+	void getDiffBetweenCommits(String priorCommit, String laterCommit);
```

Comment:
Javadoc?

Why does this return void?

Reply:
Temporary until further discussion in #87.